manage_projects.rb

```diff
@@ -66,6 +74,20 @@ def add_or_update_project(action=nil)
     {resource_groups: {:display_method => :describe_resource_groups, formatters: [formatter]}}, {filter_level: {formatters: [formatter]}}
     puts
     add_or_update_project
+  elsif action == "disable" || action == "enable"
+    print "Project name: "
+    project_name = gets.chomp
+    project = Project.find_by_name(project_name)
+    if !project
+      puts "Project not found. Please try again."
+      return add_or_update_project(action)
+    end
+    project = @factory.as_type(project)
+    if !project.end_date
```

Comment:
This means that when you choose 'enable' or 'disable' you are actually toggling the project having an end date, regardless of the choice (e.g. if select 'enable' for a project that has no end date, you will give it one). Also, it would be nice to highlight no change if trying to enable/ disable a project that is already enabled/ disabled.

Reply:
Indeed, I think this would benefit from a `disabled` attribute on `Project`.